apps/wallet-mobile/src/features/Initialization/TermsOfServiceChangedScreen/TermsOfServiceChangedScreen.tsx

```diff
@@ -44,15 +44,19 @@ export const TermsOfServiceChangedScreen = () => {
 
       <Spacer height={24} />
 
-      <BlueCheckbox checked={accepted} onPress={onPressCheckbox} style={styles.checkbox}>
+      <BlueCheckbox checked={accepted} spacing={8} onPress={onPressCheckbox} style={styles.checkbox}>
         <View style={styles.checkboxRow}>
           <Text style={styles.checkboxText}>{`${strings.tosIAgreeWith} `}</Text>
 
           <TouchableOpacity onPress={onTosLinkPress}>
             <Text style={[styles.checkboxText, styles.checkboxLink]}>{strings.tosAgreement}</Text>
           </TouchableOpacity>
 
-          <Text style={styles.checkboxText}>{` ${strings.tosAnd} `}</Text>
+          <Text style={styles.checkboxText}>{` `}</Text>
```

Comment:
I think it can be replaced by `{${utf8Symbols.NBSP}${strings.tosAnd}${utf8Symbols.NBSP}}` which translates to `\u00A0`

Reply:
quite sure it is